mllib/src/main/scala/org/apache/spark/ml/ann/Layer.scala

```diff
@@ -374,6 +380,22 @@ private[ann] trait TopologyModel extends Serializable {
   def predict(data: Vector): Vector
 
   /**
+   * Raw prediction of the model
+   *
+   * @param data input data
+   * @return raw prediction
+   */
+  def predictRaw(data: Vector): Vector
```

Comment:
Please match the Classifier API here (data -> features) unless there's a reason to deviate